src/Component/book.js

```diff
@@ -15,12 +17,30 @@ const Book = (props) => {
   };
 
   return (
-    <li className="list-item" key={id}>
-      <h4>{category}</h4>
-      <h3>{title}</h3>
-      <p>{author}</p>
-      <button type="button" onClick={removeEventHandler}>Remove book</button>
-    </li>
+    <div className="book-container">
+      <div className="list-item" key={id}>
+        <h4>{category}</h4>
+        <h3>{title}</h3>
+        <p>{author}</p>
+        <div className="btn-container">
+          <button className="commentbtn" type="button">
+            Comments
+          </button>
+          <button type="button" onClick={removeEventHandler}>Remove book</button>
+          <button className="editbtn" type="button">
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="bookProgressSection">
+        <Progress />
+        <div className="currentChapterSection">
+          <h2>CURRENT CHAPTER</h2>
+          <h3>Chapter 3: &quot;A Lesson Learned&quot;</h3>
+          <button type="button">UPDATE PROGRESS</button>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Hi @Adanzeakonobi, nice job so far. let's again make sure that styling is EXACTLY the same as that provided in Zeplin. Font size, font-weight, color, and layouts should be EXACTLY the same as that in Zeplin. 
- let's make sure the logo is aligned with the book-container as seen in the screenshot below.
- There's a vertical line that separates the Progress component and the currentChapterSection section
- Please take a keen look at the annotations on the screenshot below, compare them with the original design and make the necessary adjustments( padding, margins, font-weight)


Your Implementation | Zeplin Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/173153143-a0fd2397-417d-4cb6-b3e9-7fb4710bd9b8.png) | ![](https://user-images.githubusercontent.com/63087491/173153315-1a8fcb2c-1a13-45a4-a8e7-c94e2114c411.png)


Wish you all the best! :pray: